server/index.js

```diff
@@ -56,6 +56,103 @@ app.get('/api/products/:productId', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/cart', (req, res, next) => {
+  const sql = `
+    select "cartItemId",
+           "cartId",
+           "productId",
+           "price"
+      from "cartItems";
+  `;
+  const sqlAlt = `
+    select "c"."cartItemId",
+           "c"."price",
+           "p"."productId",
+           "p"."image",
+           "p"."name",
+           "p"."shortDescription"
+      from "cartItems" as "c"
+      join "products" as "p" using("productId")
+     where "c"."cartId" = $1
+    `;
+  if (!req.session.cartId) {
+    db.query(sql)
+      .then(result => res.json([]))
+      .catch(err => next(err));
```

Comment:
If there is no `req.session.cartId`, you should not send the client _everybody's_ cart items. 😄 Just respond with an empty array.
```js
res.json([])
```